### PR TITLE
Added UI screens for Resiliency probes

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes

image
image
image

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: chaoscenter/web/src/api/core/experiments/getExperimentRun.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  // Options
+  options = {}
+}: GqlAPIQueryRequest<
+  GetExperimentRunResponse,
+  GetExperimentRunRequest,
+  Omit<GetExperimentRunRequest, 'identifiers'>
+>): GqlAPIQueryResponse<GetExperimentRunResponse, GetExperimentRunRequest> {
+  const { data, loading, ...rest } = useQuery<GetExperimentRunResponse, GetExperimentRunRequest>(
+    gql`
+      query getExperimentRun($identifiers: IdentifiersRequest!, $experimentRunID: String, $notifyID: String) {
```
Comment (S-ayanide) [inline chaoscenter/web/src/api/core/experiments/getExperimentRun.ts:95]: We are taking `projectID` but not replacing `identifiers` with the same

File: chaoscenter/web/src/api/core/probe/getProbesInExperimentRun.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    GetProbesInExperimentRunRequest
+  >(
+    gql`
+      query GetProbesInExperimentRun($projectID: ID!, $experimentRunID: String!, $faultName: String!) {
+        getProbesInExperimentRun(projectID: $projectID, experimentRunID: $experimentRunID, faultName: $faultName) {
+          probe {
+            name
+            type
+            infrastructureType
+            kubernetesHTTPProperties {
```
Comment (S-ayanide) [inline chaoscenter/web/src/api/core/probe/getProbesInExperimentRun.ts:39]: Please add `attempt` and `evaluationTimeout` for all probes, only CMD probe has `attempt` in this implementation

File: chaoscenter/web/src/api/core/probe/listProbe.ts [hunk 3]
```diff
@@ -71,3 +140,110 @@ export function listProbes({
...
+          name
+          description
+          type
+          tags
+          type
+          updatedBy {
+            username
+            email
+          }
+          httpProperties {
```
Comment (S-ayanide) [inline chaoscenter/web/src/api/core/probe/listProbe.ts:164]: Please change thing to `kubernetesHTTPProperties`

File: chaoscenter/web/src/api/core/probe/listProbe.ts [hunk 4]
```diff
@@ -71,3 +140,110 @@ export function listProbes({
...
+            stopOnFailure
+            group
+            version
+            resource
+            namespace
+            fieldSelector
+            labelSelector
+            operation
+          }
+          cmdProperties {
```
Comment (S-ayanide) [inline chaoscenter/web/src/api/core/probe/listProbe.ts:224]: Please change this to `kubernetesCMDProperties`

File: chaoscenter/web/src/components/PipelineDiagram/Nodes/ChaosExecutionNode/NodePopovers.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          </Text>
+          <Container flex={{ justifyContent: 'flex-start' }}>
+            <Icon
+              name="execution-waiting"
+              style={{ color: `var(--orange-700)` }}
+              size={18}
+              margin={{ right: 'xsmall' }}
+            />
+            <Text font={{ variation: FontVariation.H6 }} color={Color.WHITE}>
+              NOT STARTED
```
Comment (S-ayanide) [inline chaoscenter/web/src/components/PipelineDiagram/Nodes/ChaosExecutionNode/NodePopovers.tsx:40]: Please take from strings

**Review by S-ayanide — CHANGES_REQUESTED** [review]

File: chaoscenter/web/src/views/AddProbeModalWizard/AddProbeInitialTypeContainer.tsx [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+      background={Color.PRIMARY_9}
+    >
+      <Layout.Vertical height="90%" width="50%" padding={{ left: 'xlarge' }}>
+        <div>
+          <Text font={{ variation: FontVariation.H3 }} margin={{ bottom: 'xlarge' }} color={Color.WHITE}>
+            {getString(`selectYourProbeType`)}
+          </Text>
+          <Layout.Vertical spacing={'small'}>
+            <Text font={{ variation: FontVariation.FORM_LABEL }} color={Color.WHITE}>
+              {getString('selectChaosInfrastructureType')}
```
Comment (S-ayanide) [inline chaoscenter/web/src/views/AddProbeModalWizard/AddProbeInitialTypeContainer.tsx:75]: Please comment this string too

File: chaoscenter/web/src/utils/probes.ts [hunk 7]
```diff
@@ -16,36 +24,139 @@ export function getProbeDetails(probe: ProbeAttributes): string[][] {
...
+    case ProbeType.CMD:
+      return 'Command';
+    case ProbeType.PROM:
+      return 'Prometheus';
+    case ProbeType.K8S:
+      return 'Kubernetes';
+  }
+}
+
+export function getIcon(type: ProbeType): IconName {
```
Comment (S-ayanide) [inline chaoscenter/web/src/utils/probes.ts:128]: Can we add the icons for K8S and PROM too if they are published in harnessio!

**Review by S-ayanide — CHANGES_REQUESTED** [review]

File: chaoscenter/web/src/utils/probes.ts (hunk 7 above)
Comment (Saranya-jena) [reply chaoscenter/web/src/utils/probes.ts:128]: Will need more discussion on adding it to OSS, this is for immediate solution.

**Review by S-ayanide — APPROVED** [review]
LGTM

**Pull request merged**